### DISCUSSION TITLE
How to get window's name?

### DISCUSSION THREAD

**Discussion by aurexav** [asmvik/yabai#866] [Q&A] [answered]
Some windows don't show a name on the left-top

How can I specify it in `-m rule --add app="xxx"`?

**Comment by rockyzhang24** [comment, marked as answer]
Use yabai query along with `jq`. 
E.g., to get the name the currently focused window: `yabai -m query --windows --window | jq '.title'`. 
`yabai -m query --windows --window` outputs all the fields (information) about the window in `json` format. 
`jq` can filter and output the fields that you care about. 

Some references that help you learn to use yabai:
* Its own wiki page, especially https://github.com/koekeishiya/yabai/wiki/Commands
* All the commands it supports: https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc
